### PR TITLE
Added Rule of law data

### PR DESCRIPTION
**Pull request by webbyfox**
- [x] Change has a jira ticket that has the correct status.
 - [x] Changelog entry added.

### CODE REVIEW

**Review by rafa-garcia — APPROVED** [review]
LGTM

File: dataservices/management/commands/import_rank_of_law_data.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+class Command(BaseCommand):
+    help = 'Import Income data'
+
+    def handle(self, *args, **options):
+        with open('dataservices/resources/rule_of_law_rank.csv', 'r', encoding='utf-8-sig') as f:
+            data = tablib.import_set(f.read(), format='csv', headers=True)
+            ruleoflaw_data = []
+
+            for item in data:
```
Comment (depsiatwal) [inline dataservices/management/commands/import_rank_of_law_data.py:15]: tests please to cover each of these flows

**Review by depsiatwal — COMMENTED** [review]
small comment about tests else LGTM

**Comment by webbyfox** [comment]
Added missing test for management command.

**Review by depsiatwal — APPROVED** [review]
lgtm

**Pull request merged**